src/python/platforms/fuchsia/util/device.py

```diff
@@ -193,13 +193,13 @@ def ls(self, path):
     try:
       p = self._ssh(['ls', '-l', path], stdout=subprocess.PIPE).popen()
       out, _ = p.communicate()
-      for line in str(out).split('\n'):
+      for line in out.decode('utf-8').split('\n'):
         # Line ~= '-rw-r--r-- 1 0 0 8192 Mar 18 22:02 some-name'
         parts = line.split()
         # When we're running ls over ssh, we may get a note about
         # "Warning: Permanently added [address] to the list of known hosts"
         # Don't try to treat those as file paths
-        if len(parts) > 8 and "Warning:" not in parts:
+        if len(parts) > 8 and 'Warning:' not in parts:
```

Comment:
b'Warning:' ?

Reply:
parts is a str (unicode) so that would be wrong here.